src/sentry/api/endpoints/organization_config_repositories.py

```diff
@@ -11,11 +11,6 @@ class OrganizationConfigRepositoriesEndpoint(OrganizationEndpoint):
 
     def get(self, request, organization):
         provider_bindings = bindings.get('repository.provider')
-        integrations_provider_bindings = bindings.get('integration-repository.provider')
-        has_catchall = features.has(
-            'organizations:internal-catchall',
-            organization,
-            actor=request.user)
         has_github_apps = features.has('organizations:github-apps',
```

Comment:
@MeredithAnya does `has_github_apps` still need to be here?

Reply:
Hmmm no. Since we aren't going to allow adding repos from the old UI for the new integrations you don't need it.